components/keyboard_shortcuts/keyboard_shortcuts_sequence/keyboard_shortcuts_sequence.test.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {mountWithIntl} from 'tests/helpers/intl-test-helper';
+
+import KeyboardShortcutsSequence from './keyboard_shortcuts_sequence';
+
+describe('components/shortcuts/KeyboardShortcutsSequence', () => {
```

Comment:
can we add some tests for the hoisting logic etc? We are trying to get as much coverage in unit tests as possible :)